### ISSUE TITLE
[BUG] err="function not implemented"

### ISSUE THREAD

**Issue by fffmonkeyking** [occlum/occlum#1339]
# Describe the bug
go_app.go source:
... ...
import (
	... ...
	"github.com/fsnotify/fsnotify"
)
... ...
watcher, err := fsnotify.NewWatcher()
if err != nil {
    log.Error("Failed to start filesystem watcher", "err", err)
    return
}
...

run go_app: occlum run /bin/go_app
but report error:
Failed to start filesystem watcher       err="function not implemented"

--------------------------------------------------------------------------------
I briefly analyzed the execution process of fsnotify. NewWatcher()：
fsnotify. NewWatcher() --->
github.com\fsnotify\fsnotify@v1.6.0\backend_inotify.go's      NewWatcher() --->
golang.org/x/sys@v0.5.0/unix/zsyscall_linux.go's     InotifyInit1(unix.IN_CLOEXEC | unix.IN_NONBLOCK) --->
    r0, _, e1 := RawSyscall(SYS_INOTIFY_INIT1, uintptr(flags), 0, 0)

So, Does occlum or occlum's libos not implement the system call for SYS_INOTIFY_INIT1 ???

# To reproduce
1. 
2. 
3. 
4. 

# Expected behavior

# Logs

# Environment

- HW: [SGX2] 
- OS: [Ubuntu20.04]
- Occlum version: [occlum/occlum:0.29.6-ubuntu20.04]

# Additional context

# Possible solution/Implementation
1、Where can I find a list of system calls that Occlum or Occlum's libos does not implement ？

**Comment by Antone077** [comment]
@fffmonkeyking yes ,Occlum does not implement InotifyInit1(), you can find in this file(https://github.com/occlum/occlum/blob/master/src/libos/src/syscall/mod.rs#L391C58-L391C58)
